broker-util/oo-admin-chk

```diff
@@ -171,7 +171,8 @@ node_hash.each do |gear_uuid, gear_info|
     if !datastore_has_gear?(gear_uuid)
       puts "#{gear_uuid}...FAIL" if $verbose
       print_message "Gear #{gear_uuid} exists on node #{gear_info[0]} (uid: #{gear_info[1]}) but does not exist in mongo database"
-      print_message "Please see https://access.redhat.com/solutions/1171163 for more information."
+      print_message "To fix this issue, remove the gear by using the oo-devel-node command:"
```

Comment:
One tiny nit-pick, lets be specific about where this command should be run. Users will run oo-admin-chk on a broker, but will need to run oo-devel-node on which node the gear is installed on. If we have the `server_identity` we should point the user there. Otherwise, just let them know it needs to be run on the node.